zappa_settings.example.json

```diff
@@ -0,0 +1,24 @@
+{
+    "dev": {
+        "app_function": "rickroll.app",
+        // Put a random S3 bucket name that doesn't already exist here. Zappa
+        // will create it and put your Lambda code in it when you run
+        // 'zappa init'.
+        "s3_bucket": "insert-bucket-name-here",
```

Comment:
One of my least favourite things about JSON is that it doesn't allow comments...  😭 

While Zappa may be relaxed enough to allow them, the QA/Testing part of my soul stays awake at night when I know we're technically breaking a format standard.

I know it's awful, but can I ask for something like:

```json
"s3_bucket": "An S3 bucket that doesn't already exist. Zappa will create it and use it for storage when you run `zappa init`"
```

Comments are *waaay* nicer, and if we were using YAML or [HJSON](https://hjson.org/) I'd be all over them. But every linter (including github's changes view) and my psyche is going to worry every time it sees a `//` otherwise. :/